src/java/picard/analysis/CollectRnaSeqMetrics.java

```diff
@@ -49,12 +49,52 @@
 import java.util.Set;
 
 @CommandLineProgramProperties(
-        usage = "Collect metrics about the alignment of RNA to various functional classes of loci in the genome:" +
-                "coding, intronic, UTR, intergenic, ribosomal. Also determines strand-specificity for strand-specific libraries.",
-        usageShort = "Produces RNA alignment metrics for a SAM or BAM file",
+        usage = CollectRnaSeqMetrics.USAGE_SUMMARY + CollectRnaSeqMetrics.USAGE_DETAILS,
+        usageShort = CollectRnaSeqMetrics.USAGE_SUMMARY,
         programGroup = Metrics.class
 )
 public class CollectRnaSeqMetrics extends SinglePassSamProgram {
+
+    static final String USAGE_SUMMARY = "Produces RNA alignment metrics for a SAM or BAM file.  ";
+    static final String USAGE_DETAILS = "<p>This tool takes a SAM/BAM file containing aligned RNA transcripts and produces metrics describing the distribution " +
+            "of the bases within the transcripts.  Calculates the total numbers and the percentages of nucleotides within the transcript sub-domains" +
+            " including the untranslated region (UTR), introns, intergenic sequences (between discrete genes) and peptide-coding sequences.  " +
+            "This tool also determines the numbers of bases that pass the sequencing vendor's quality filters (PF_BASES). " +
+            "To learn more about the PF metric, please see the corresponding GATK " +
+            "<a href='https://www.broadinstitute.org/gatk/guide/article?id=6329'><b>dictionary entry</b></a>.</p>" +
+            "" +
+            "<p>Other metrics include the median coverage (depth), 5'/3'-biases, and the numbers of reads with the correct/incorrect " +
+            "strand designation. The 5'/3'-biases represent the bias introduced by reverse transcriptase enzymes during library construction, which" +
+            " result in the over representation of either the 5 or 3' ends of transcripts.  Please see the following " +
+            "<a href=\"https://www.biostars.org/p/102812/#102815\"><b>link</b></a> for information on how this ratio is calculated. </p>" +
+            "" +
+            "<p>The input must be a valid BAM file containing RNAseq data aligned using e.g. <a href=\"http://github.com/alexdobin/STAR\"><b>STAR</b></a>" +
```

Comment:
I considered moving the cmdline example to the quickstart page but was concerned that it would make the page too long. I figure users will need to consult the tool doc page at least once when they start, so I think that's still a good place to explain how to invoke tools. It's appropriately topical.

I agree it would be convenient to have a menu of links. Making a toolbar listing all the pages at the top should be easy since there are not many.